paws/values.yaml

```diff
@@ -119,7 +119,9 @@ jupyterhub:
             return pod
 
           c.KubeSpawner.modify_pod_hook = fix_labels
-      myConfig: |
+      00-myConfig: |
+          localdev = True
```

Comment:
nit: As I understand, this would be the "default" value for it right? Would it be better to make it the prod one (`localdev = False`), to avoid ever deploying dev in prod by mistake? (deploying prod in dev is less dangerous)

Reply:
In practice this could be done, but it would require more than switching here. localdev is used in other contexts, true is the default, it is overridden by `-f paws/production.yaml` fed to helm.

In this situation I would argue we should keep it this way, as secrets also has to be fed to prod for it to work, so it looks more like `helm upgrade paws --namespace prod ./paws -f paws/secrets.yaml -f paws/production.yaml --timeout=50m` (https://wikitech.wikimedia.org/wiki/PAWS/Admin#Deployment_2)

By keeping the status quo we are able to keep the install instructions more concise for any new user that may be interested in contributing.

Reply:
> By keeping the status quo we are able to keep the install instructions more concise for any new user that may be interested in contributing.

That can be eased with scripts and autodetection, something similar to https://wikitech.wikimedia.org/wiki/Wikimedia_Cloud_Services_team/EnhancementProposals/Decision_record_T303931_k8s_standard_deployment_code_pattern (that we might want to adopt here too at some point), that will simplify the install instructions even more.

In any case, out of scope of this PR if it involves more changes than just here.

Reply:
Seems reasonable. Indeed this is a place to introduce the startup script standard that has been mentioned.
https://phabricator.wikimedia.org/T322303
https://phabricator.wikimedia.org/T322304